homekit/AirConditioner.js

```diff
@@ -1,36 +1,47 @@
+// eslint-disable-next-line no-unused-vars
+const homebridge = require('homebridge')
+// eslint-disable-next-line no-unused-vars
```

Comment:
This is required for the type checking in VS Code to work unfortunately. Otherwise you can't define the below (for example):

```
/** @type {typeof homebridge.Service} */
this.Service = platform.api.hap.Service
```